From 28ad76b51587d97a127b7e00803c6a3ee0c9defb Mon Sep 17 00:00:00 2001
From: Venji10 <38562746+Venji10@users.noreply.github.com>
Date: Fri, 28 Jun 2019 22:09:34 +0200
Subject: [PATCH] Update moduleparam.h

---
 include/linux/moduleparam.h | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/include/linux/moduleparam.h b/include/linux/moduleparam.h
index e43253372461..bfe69b50945c 100644
--- a/include/linux/moduleparam.h
+++ b/include/linux/moduleparam.h
@@ -161,12 +161,6 @@ struct kparam_array
  *
  * The ops can have NULL set or get functions.
  */
-#define module_param_cb(name, ops, arg, perm)				      \
-	__module_param_call(MODULE_PARAM_PREFIX, name, ops, arg, perm, -1, 0)
-
-#define module_param_cb_unsafe(name, ops, arg, perm)			      \
-	__module_param_call(MODULE_PARAM_PREFIX, name, ops, arg, perm, -1,    \
-			    KERNEL_PARAM_FL_UNSAFE)
 
 /**
  * <level>_param_cb - general callback for a module/cmdline parameter
