From 35f553645648a363869701078abafba2f2a03a17 Mon Sep 17 00:00:00 2001
From: Venji10 <38562746+Venji10@users.noreply.github.com>
Date: Sat, 22 Jun 2019 15:41:14 +0200
Subject: [PATCH] Update kgsl_events.c

---
 drivers/gpu/msm/kgsl_events.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/msm/kgsl_events.c b/drivers/gpu/msm/kgsl_events.c
index 759a96601170..2effdc347045 100644
--- a/drivers/gpu/msm/kgsl_events.c
+++ b/drivers/gpu/msm/kgsl_events.c
@@ -15,7 +15,7 @@
 #include <linux/list.h>
 #include <linux/workqueue.h>
 #include <linux/debugfs.h>
-#include <kgsl_device.h>
+#include "kgsl_device.h"
 
 #include "kgsl_debugfs.h"
 #include "kgsl_trace.h"
