From 42b75763ef617a3ac14e6cd29207c145d5a56f56 Mon Sep 17 00:00:00 2001
From: Venji10 <38562746+Venji10@users.noreply.github.com>
Date: Sun, 23 Jun 2019 13:37:10 +0200
Subject: [PATCH] Update define_trace.h

---
 include/trace/define_trace.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/trace/define_trace.h b/include/trace/define_trace.h
index 6e3945f64102..0172ba5a01de 100644
--- a/include/trace/define_trace.h
+++ b/include/trace/define_trace.h
@@ -85,7 +85,7 @@
 /* Let the trace headers be reread */
 #define TRACE_HEADER_MULTI_READ
 
-#include TRACE_INCLUDE(TRACE_INCLUDE_FILE)
+#include "../../drivers/media/platform/msm/camera/cam_utils/cam_trace.h"
 
 /* Make all open coded DECLARE_TRACE nops */
 #undef DECLARE_TRACE
