From 558f83819d68f4ea5da9945fcf205c89cc2a6086 Mon Sep 17 00:00:00 2001
From: Venji10 <38562746+Venji10@users.noreply.github.com>
Date: Sun, 23 Jun 2019 16:21:28 +0200
Subject: [PATCH] Update configfs.c

---
 drivers/usb/gadget/configfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/gadget/configfs.c b/drivers/usb/gadget/configfs.c
index 10c6234fff4a..a5372c5b7c46 100644
--- a/drivers/usb/gadget/configfs.c
+++ b/drivers/usb/gadget/configfs.c
@@ -16,7 +16,7 @@
 #include <linux/usb/ch9.h>
 
 #ifdef CONFIG_USB_F_NCM
-#include <function/u_ncm.h>
+#include "function/u_ncm.h"
 #endif
 
 #ifdef CONFIG_USB_CONFIGFS_F_ACC
