From 83b0ace09f44f51477b91fb8184fc89efd38d108 Mon Sep 17 00:00:00 2001
From: Venji10 <38562746+Venji10@users.noreply.github.com>
Date: Sun, 23 Jun 2019 16:01:33 +0200
Subject: [PATCH] Update idtp9220.c

---
 drivers/power/supply/qcom/idtp9220.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/power/supply/qcom/idtp9220.c b/drivers/power/supply/qcom/idtp9220.c
index 2b2a11921171..fbb6b5d54b7d 100644
--- a/drivers/power/supply/qcom/idtp9220.c
+++ b/drivers/power/supply/qcom/idtp9220.c
@@ -20,7 +20,7 @@
 #include <linux/workqueue.h>
 #include <linux/sysfs.h>
 #include <asm/unaligned.h>
-#include <idtp9220.h>
+#include "idtp9220.h"
 #include <linux/regmap.h>
 #include <linux/spinlock.h>
 #include <linux/of_gpio.h>
