From e1c870c37e29dc4e65b52251a8536c2c26a92663 Mon Sep 17 00:00:00 2001
From: Venji10 <38562746+Venji10@users.noreply.github.com>
Date: Sat, 22 Jun 2019 12:44:31 +0200
Subject: [PATCH] Update btfm_slim_wcn3990.c

---
 drivers/bluetooth/btfm_slim_wcn3990.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/bluetooth/btfm_slim_wcn3990.c b/drivers/bluetooth/btfm_slim_wcn3990.c
index 661518935086..59b23a24dd81 100644
--- a/drivers/bluetooth/btfm_slim_wcn3990.c
+++ b/drivers/bluetooth/btfm_slim_wcn3990.c
@@ -10,7 +10,7 @@
  * GNU General Public License for more details.
  */
 #include <linux/slimbus/slimbus.h>
-#include <btfm_slim.h>
+#include "btfm_slim.h"
 #include <btfm_slim_wcn3990.h>
 
 /* WCN3990 Port assignment */
